README.md

```diff
@@ -1 +1,26 @@
 # Scope for Maven Action
+
+### Easy integration with Scope for Java projects based on Maven
+
+>The latest release of this Action is based on Scope Jave Agent v0.2.2
```

Comment:
We can remove this line, as we don't want to be updating it constantly.

Reply:
Fixed